source/includes/entityref/_jobshift.md

```diff
@@ -0,0 +1,164 @@
+# JobShift
+
+JobShift is the main entity for shift scheduling and is used for storing start & end times, references to the clientCorporation, job, and other details for the shift (such as location, name, etc).
```

Comment:
Change to something like:
The main entity for shift scheduling. Used for storing start and end times, references to the ClientCorporation, JobOrder, and other details for the shift such as location, name, and so forth.

Throughout the doc, use the real camelcased entity name where appropriate.